### ISSUE TITLE
First tag loaded has spaces removed.

### ISSUE THREAD

**Issue by trevorhreed** [iog3/SOTag#25]
For some reason, the first space in the first tag is removed when using this plugin.  See screenshots for example:

![sotag-screenshot-1](https://cloud.githubusercontent.com/assets/3343902/9067553/61ca54ae-3a9b-11e5-941e-38eca86aea3d.PNG)

And the markup looks like this:

![sotag-screenshot-2](https://cloud.githubusercontent.com/assets/3343902/9067603/a5406278-3a9b-11e5-9e49-c6ebc1cceae9.png)

You can see clearly that the input value contains a space between "Sam" and "Payne", however the tag does not.

**Comment by trevorhreed** [comment]
Oddly enough, if I append a space to the end of the tag value (e.g. "b07fbb53-00c8-4f8b-96c2-9918d0bbad70_Sam Payne ", then the tag displays as it should and the space is not removed.

**Comment by iog3** [comment]
Thanks for the report I will take a look in a few hours

**Comment by trevorhreed** [comment]
Thanks. Said workaround is sufficient for us for now, but I thought you'd like to know.

**Comment by iog3** [comment]
Do you have a live version I can play with? I can't seem to replicate it on my local. I assume the `b07fbb53-00c8-4f8b-96c2-9918d0bbad70` is the primary key for the tags table?

**Comment by trevorhreed** [comment]
That's correct, that is the primary key. Unfortunately, I don't have an accessible version that you can play with. However, I just checked the version number; we're using version 1.0.0 (12.2013), so it's entirely possible this isn't an issue in the latest version. When I get a chance, I'll try plugging in the latest version to see if that resolves the issue.

**Comment by iog3** [comment]
Have a look at the dev branch, it has the latest code although it's
probably buggy, I've been making changes the last few days and have a few
more todo before another release.

I don't remember fixing anything like your issue so it's probably still an
issue :(

On 4 Aug 2015, at 8:35 pm, Trevor <notifications@github.com> wrote:

That's correct, that is the primary key. Unfortunately, I don't have an
accessible version that you can play with. However, I just checked the
version number; we're using version 1.0.0 (12.2013), so it's entirely
possible this isn't an issue in the latest version. When I get a chance,
I'll try plugging in the latest version to see if that resolves the issue.

—
Reply to this email directly or view it on GitHub
<https://github.com/iog3/SOTag/issues/25#issuecomment-127730216>.
